pylibdmtx/wrapper.py

```diff
@@ -397,3 +463,37 @@ def libdmtx_function(fname, restype, *args):
     DmtxPassFail,
     POINTER(POINTER(DmtxRegion))
 )
+
+dmtxImageGetProp = libdmtx_function(
+    'dmtxImageGetProp',
+    DmtxPassFail,
```

Comment:
Return type is `int` in my copy of `dmtx.h` (version 0.7.4). I would expect `int` rather than `DmtxPassFail` given that this function returns value of a property.

Reply:
correct, I probably looked into SetProp by mistake